.papr.sh

```diff
@@ -1,11 +1,10 @@
 #!/bin/bash
 set -xeuo pipefail
+export GOPATH=/go
+export PATH=$HOME/gopath/bin:$PATH:$GOPATH/bin
+export GOSRC=$GOPATH/src/github.com/containers/libpod
```

Comment:
libpod or buildah?

Reply:
I like both, but it should be buildah. Thanks for checking!